### PR TITLE
Update ember-cli-build.js

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: webapp/ember-cli-build.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = function(defaults) {
           'handlebars.js': null
       },
       'ember-cli-babel': {
-          includePolyfill: not_testing,
```
Comment (stefanpenner) [inline webapp/ember-cli-build.js]: looks like you had it explicitly disabled in testing?

Comment (stefanpenner) [inline webapp/ember-cli-build.js]: This is likely for a good, reason although Im not sure what that reason is.

Comment (vmalloc) [inline webapp/ember-cli-build.js]: I was following advice from another thread due to issues I stumbled upon: https://slack-redir.net/link?url=https%3A%2F%2Fgithub.com%2Fsamselikoff%2Fember-cli-mirage%2Fissues%2F1084. Before I did that, I had lots of weirder errors

Comment (vmalloc) [inline webapp/ember-cli-build.js]: @stefanpenner for example https://travis-ci.org/getslash/backslash/builds/232506426#L6052

Comment (stefanpenner) [inline webapp/ember-cli-build.js]: well without it, you cannot use stuff like `startsWith`.

**Comment by vmalloc** [comment]
Already changed manually. Thanks for the help!

**Pull request closed without merging**